src/supply.ts

```diff
@@ -1,115 +1,49 @@
 import { connect } from './providers'
 import { contracts } from './constants'
+import { BigNumber, Contract, constants } from 'ethers'
+import { formatUnits } from 'ethers/lib/utils'
 
 const [network, provider, wallet, contractAt] = connect()
 
-// calculate circulating supply
-export const getCirculatingSupply = async () => {
-    const burned = await getBurned()
-    const teamCirculating = await getTeamCirculating()
-    const futureTeamCirculating = await getFutureTeamCirculating()
-    const incentiveCirculating = await getIncentiveCirculating()
-    const companyCirculating = await getCompanyCirculating()
-    const preSale = await getPreSale()
-    const protocolBalances = await getProtocolBalances()
-
-    console.log('burned: '+burned)
-    console.log('teamCirculating: '+teamCirculating)
-    console.log('futureTeamCirculating: '+futureTeamCirculating)
-    console.log('incentiveCirculating: '+incentiveCirculating)
-    console.log('companyCirculating: '+companyCirculating)
-    console.log('preSale: '+preSale)
-    console.log('protocolBalances: '+protocolBalances)
-
-    const circulatingSupply = -burned+teamCirculating+futureTeamCirculating+incentiveCirculating+companyCirculating+preSale-protocolBalances
-    return circulatingSupply
-}
-
-// calculate total supply through tru contract
-const getTotalSupply = async () => {
+const LOCKED_SUPPLY_ADDRESSES = [
+    contracts.protocolDaoTreasury,
+    contracts.incentivesDaoTreasury,
+    contracts.stakingDistributor,
+    contracts.stkTruDistributor,
+    contracts.liquidityGaugeDistributor,
+    contracts.nxmDistributor,
+    contracts.options,
+    contracts.futureTeam,
+    contracts.preSaleUnclaimedMultiSig,
+    contracts.preSaleRegisteredUnclaimed,
```

Comment:
Do we have some list of addresses holding `LOCKED_SUPPLY` just to double check 😃